docstore/memdocstore/urls.go

```diff
@@ -37,7 +37,11 @@ const Scheme = "mem"
 // The URL's host is the name of the collection.
 // The URL's path is used as the keyField.
 //
-// No query parameters are supported.
+// The revision_field query parameter specifies the name of the revision field.
```

Comment:
Just want to be consistent with other provider packages:

```go
// The following query parameters are supported:
//
//   - revision_field:
//   - filename:
```

Reply:
done